scripts/upload/add_time_series_pits_2021.py

```diff
@@ -19,13 +19,13 @@ def main():
     """
     Snowex 2021 timeseries pits
     """
-    db_name = 'localhost/snowex'
+    db_name = 'localhost/test'
```

Comment:
Looks like this is no longer needed with everything coming from the `credentials.json` file